DS/DSassignment1/DoubleeNode.java

```diff
@@ -0,0 +1,259 @@
+
+public class DoubleeNode<T> 
+{
+	
+	T nodeValue;
+
+	DoubleeNode<T> start;
+	DoubleeNode<T> next;
+	DoubleeNode<T> previous;
+   
+	public DoubleeNode() {
+		nodeValue = null;
+		next = null;
+		previous = null;
+	}
+	
+	public DoubleeNode(T item)
+	{
+		nodeValue = item;
+		next = null;
+		previous = null;
+	}
+	
+	boolean addElement(T element)
```

Comment:
ACTION: This method should not be a part of this class as a "node" does not know how another node is added to it. Better there be another class may be "DoubleeList" which has this or such other method in it, and 'DoubleeNode' can be a nested class of it (may be a private class)